wye/workshops/mixins.py

```diff
@@ -38,6 +41,63 @@ def dispatch(self, request, *args, **kwargs):
         return super(WorkshopFeedBackMixin, self).dispatch(request, *args, **kwargs)
 
 
+class WorkshopRestrictMixin(object):
+    """
+    Mixin to restrict
+        - For organisation to add workshop if no feedback is shared.
+        - For presenter to takeup workshop if no feedback is shared
+    """
+
+    def dispatch(self, request, *args, **kwargs):
+        self.user = request.user
+        self.feedback_required = []
+        usertype = self.user.profile.usertype
+
+        # check if user is tutor
+        if usertype.filter(display_name__icontains="tutor").exists():
```

Comment:
@shankisg : I am Unable to get the reason for checking usertype with name. Can't **slug** field with **==** be fine here? Thanks!